src/language-css/parser-postcss.js

```diff
@@ -120,6 +120,23 @@ function parseValueNode(valueNode, options) {
       parenGroupStack.pop();
       parenGroup = parenGroupStack[parenGroupStack.length - 1];
     } else if (node.type === "comma") {
+      if (commaGroup.groups.length > 1) {
+        for (const group of commaGroup.groups) {
+          // if css interpolation
+          if (
+            group.value &&
+            typeof group.value === "string" &&
+            group.value.includes("#{")
+          ) {
+            commaGroup.groups = [
+              stringifyNode({
+                groups: commaGroup.groups,
+              }).trim(),
```

Comment:
Not sure it is safe to trim here, why we do it?

Reply:
Stringify was adding extra '\n', so I am using trim